lib/iris/coord_systems.py

```diff
@@ -840,17 +842,33 @@ def __init__(self, longitude_of_projection_origin=0, ellipsoid=None):
                     True longitude of planar origin in degrees.
             * ellipsoid
                     :class:`GeogCS` defining the ellipsoid.
+            * min_latitude
+                    the maximum southerly extent of the projection.
+                    Defaults to -80 degrees.
+            * max_latitude
+                    the maximum northerly extent of the projection.
+                    Defaults to 84 degrees.
+            * true_scale_lat
+                    the latitude where the scale is 1. Defaults to 0 degrees.
 
         """
-
-        #: True longitude of planar origin in degrees.
         self.longitude_of_projection_origin = longitude_of_projection_origin
-        #: Ellipsoid definition.
         self.ellipsoid = ellipsoid
+        self.min_latitude = min_latitude
+        self.max_latitude = max_latitude
+        self.true_scale_lat = true_scale_lat
 
     def __repr__(self):
-        res = "Mercator(longitude_of_projection_origin={!r}, ellipsoid={!r})"
-        return res.format(self.longitude_of_projection_origin, self.ellipsoid)
+        res = ("Mercator(longitude_of_projection_origin={!r}, "
```

Comment:
I'm completely happy with your implementation. FWIW, if I was implementing this though, I'd have done:

```"Mercator(longitude_of_projection_origin={self.longitude_of_projection_origin!r}...)".format(self=self)```

Reply:
Yeah, it was line length that put me off doing that...

Reply:
Also, I think that's actually less readable than having unnamed args in the format string.

Reply:
> Also, I think that's actually less readable than having unnamed args in the format string.

That horse has bolted I'm afraid... https://www.python.org/dev/peps/pep-0498/
😜

Reply:
> That horse has bolted

Yes yes I know, and it makes full sense in the case of full format strings (i.e. `f''`). I just don't much like this intermediate situation 😉